warehouse/static/sass/resets/_normalize.scss

```diff
@@ -134,14 +134,6 @@ strong {
   font-weight: bold;
 }
 
-/**
- * Address styling not present in Safari and Chrome.
- */
-
-dfn {
-  font-style: italic;
```

Comment:
We probably want to explicitly set this to `font-style: normal;` if the default is for this to become italic in some browsers.

Reply:
@jpurviance Can you resolve this?